datastore.nimble

```diff
@@ -10,10 +10,10 @@ requires "nim >= 1.2.0",
          "asynctest >= 0.5.2 & < 0.6.0",
          "chronos >= 4.0.3 & < 5.0.0",
          "questionable >= 0.10.15 & < 0.11.0",
-         "sqlite3_abi",
-         "leveldbstatic >= 0.1.6",
-         "stew",
-         "unittest2"
+         "sqlite3_abi == 3.47.0.0",
+         "leveldbstatic >= 0.2.0 & < 1.0.0",
```

Comment:
I think a lot of nim libs use the second digit as a major version indicator 🤷 It should not be the case imo.
```suggestion
         "leveldbstatic >= 0.2.0 & < 0.3.0",
```